### PR TITLE
1st (not really) balance tweaks

### LINKED ISSUE #1604: Claimant faction in evil governments
**Describe your suggestion in full detail below:**
Allow claimant factions in evil governments.

### LINKED ISSUE #1605: Limited Great War
**Describe your suggestion in full detail below:**
Limit Great War CB used by the Horde, Legion, Scourge, and Black Empire to the kingdom tier (right now it annex the whole realm), but make it cheaper. Thus their conquests should take more time.

### LINKED ISSUE #1606: Ruled too long penalty
**Describe your suggestion in full detail below:**
So, the Paradox planned to add Ruled too long penalty to counter immortal rulers, but it doesn't work both in the vanilla and mod. I suggest adding it back.
P.S. There are events for immortals in `rip_flavor_events`. They need adaptation together with the suggestion.

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Undead Horde, Demonic Legion, Black Pact governments can start Claimant factions against their lieges.
- Great War CB takes titles in the selected de-jure kingdom instead of the whole realm, but costs cheaper now.
- Undead (modifier), demonic and Shath'yar (cultures) provinces can't get depopulation from diseases.
- Rulers get -15 Vassal Opinion and -15% Plot Power Defence per 50 ruled years. This should make immortal realms more dynamic. This comes with a game rule to regulate who gets the penalty.

## Developer changelog:
- "Adapted" immortal events. Actually, commented them since no one hides immortality in Warcraft.

# How to test:

Closes #1604, closes #1605, closes #1606.

### CODE REVIEW

**Comment by zumbak04** [comment]
> I cant add a claim to myself or other chars with the console for a faction to elevate a claimaing to the scourge and burning legion empire tittles

Now vassals can claim these titles even without claims.

**Comment by zumbak04** [comment]
> Something new happens now, it seems even if you present the ultimatum to the LK, and he refuses, nothing happens. You dont go to war against him

Fixed in 53176fc3e294d2ec12292fb4ee9f95cc0a8bb242.

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2978,6 +2978,17 @@ GetRegenerationReason_Loa;Due to Loa's blessings;;Wegen der Segen der Loa;;;;;;;
...
+WAR_NAME_TOTAL_INVASION_OF;$ORDER$$FIRST_ADJ$ Great Invasion of $OTHER_TITLE$;;$ORDER$$FIRST_ADJ$ Gro�e Invasion von $OTHER_TITLE$;;;;;;;;;;;x
+EVTDESC_WCIMM_20;You notice that your courtiers speak badly of you more often lately. No wonder, for you have ruled for an unusually long time. Both you and your people have grown weary of your rule.;;Ihr stellt fest, dass Eure H�flinge in letzter Zeit vermehrt schlecht �ber Euch sprechen. Kein Wunder, denn Ihr regiert schon au�ergew�hnlich lange. Sowohl Ihr als auch Euer Volk sind Eurer Herrschaft �berdr�ssig geworden.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:2982]:
```suggestion
EVTDESC_WCIMM_20;You have noticed that your courtiers are speaking badly of you more often lately. It's no wonder, for you have ruled for an unusually long time. Both you and your people have grown weary of your rule.;;Ihr stellt fest, dass Eure H�flinge in letzter Zeit vermehrt schlecht �ber Euch sprechen. Kein Wunder, denn Ihr regiert schon au�ergew�hnlich lange. Sowohl Ihr als auch Euer Volk sind Eurer Herrschaft �berdr�ssig geworden.;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -2978,6 +2978,17 @@ GetRegenerationReason_Loa;Due to Loa's blessings;;Wegen der Segen der Loa;;;;;;;
...
+EVTOPTA_WCIMM_20;Years go by, nothing changes.;;Die Jahre vergehen, doch nichts �ndert sich.;;;;;;;;;;;x
+ruled_too_long_modifier;Ruled Too Long;;Zu lange Herrschaft;;;;;;;;;;;x
+RULE_RULED_TOO_LONG_PENALTY;�RWarCraft�!: Ruled Too Long Penalty;;�RWarCraft�!: Strafe f�r zu lange Herrschaft;;;;;;;;;;;x
+RULE_RULED_TOO_LONG_PENALTY_ON;On;;Aktiviert;;;;;;;;;;;x
+RULE_RULED_TOO_LONG_PENALTY_ON_DESC;Rulers get �R-15 Vassal Opinion�! and �R-15% Plot Power Defence�! per 50 years ruled.;;Herrscher erhalten �R-15 Vassalenmeinung�! und �R-15% Verschw�rungs-Aufdeckchance�! pro 50 regierten Jahren.;;;;;;;;;;;x
+RULE_RULED_TOO_LONG_PENALTY_AI;Only AI;;Nur KI;;;;;;;;;;;x
+RULE_RULED_TOO_LONG_PENALTY_AI_DESC;Only AI rulers get �R-15 Vassal Opinion�! and �R-15% Plot Power Defence�! per 50 years ruled.;;Nur KI-Herrscher erhalten �R-15 Vassalenmeinung�! und �R-15% Verschw�rungs-Aufdeckchance�! pro 50 regierten Jahren.;;;;;;;;;;;x
+RULE_RULED_TOO_LONG_PENALTY_OFF;Off;;Deaktiviert;;;;;;;;;;;x
+RULE_RULED_TOO_LONG_PENALTY_OFF_DESC;The penalty for long time rule is disabled.;;Die Strafe f�r zu lange Herrschaft ist deaktiviert.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:2991]:
```suggestion
RULE_RULED_TOO_LONG_PENALTY_OFF_DESC;The penalty for ruling too long is disabled.;;Die Strafe f�r zu lange Herrschaft ist deaktiviert.;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv (hunk 1 above)
Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv]: Done in 1bfabf901ae20d7b320846f02dcec1de51a0c4a6.

**Pull request merged**